trunner/target/armv7m4.py

```diff
@@ -46,9 +64,16 @@ def __init__(self, harness: Callable[[TestResult], TestResult]):
 
     def __call__(self, result: TestResult) -> TestResult:
         # Set of config parameters used in Openocd to flash up stm32l4a6
-        openocd_args = ["-c", "reset_config srst_only srst_nogate connect_assert_srst", "-c", "init;reset"]
-
-        with OpenocdGdbServer(interface="stlink", target="stm32l4x", extra_args=openocd_args).run():
+        openocd_args = [
+            "-c",
+            "reset_config srst_only srst_nogate connect_assert_srst",
+            "-c",
+            "init;reset",
+        ]
+
+        with OpenocdGdbServer(
+            interface="stlink", target="stm32l4x", extra_args=openocd_args
+        ).run():
```

Comment:
It not needed here, but for the future, I it might be better to separate code formatting changes from actual code changes into a separate commit, as it could improve the readability of changes/review